package.json

```diff
@@ -50,38 +50,43 @@
     ]
   },
   "dependencies": {
-    "@babel/eslint-parser": "^7.24.8",
     "@bpmn-io/properties-panel": "^3.22.3",
-    "@material-ui/styles": "^4.11.5",
+    "@emotion/react": "^11.13.0",
+    "@emotion/styled": "^11.13.0",
     "@monaco-editor/react": "^4.6.0",
+    "@mui/material": "^5.16.5",
```

Comment:
Why are we still using Material UI? I can't find a single reference to this library in the code base